pom.xml

```diff
@@ -103,32 +106,48 @@
         <dependency>
             <groupId>org.neo4j</groupId>
             <artifactId>neo4j</artifactId>
-            <version>2.2.1</version>
+            <version>2.3.0-M02</version>
```

Comment:
I wouldn't make version changes within the same commit, it makes harder to debug possible problem afterwards